src/System.Buffers/src/Resources/Strings.resx

```diff
@@ -117,4 +117,25 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="InvalidDoubleReturn" xml:space="preserve">
+    <value>The specified Buffer has already been returned to the pool.</value>
+  </data>
```

Comment:
You don't have this event currently. It's up to you, but you might want to remove this message for now.